aten/src/ATen/core/TensorBase.h

```diff
@@ -203,10 +203,7 @@ class TORCH_API TensorBase {
     impl_.reset();
   }
 
-  TensorBase& operator=(const TensorBase& x) & {
-    impl_ = x.impl_;
-    return *this;
-  };
+  TensorBase& operator=(const TensorBase& x) & = default;
   TensorBase& operator=(TensorBase&& x) & noexcept {
```

Comment:
This is equivalent to default assignment too, it's just explicitly noexcept

Reply:
I have reverted the default version. Maybe a MSVC bug.